src/conditionals/programs_with_decision_points/medium/q6/MainTest.java

```diff
@@ -0,0 +1,88 @@
+package conditionals.programs_with_decision_points.medium.q6;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+public class MainTest extends BaseTest {
+  public Clause[] testSentence() {
```

Comment:
After the class declaration, we always add a tag so we know what type of question this is testing. In this case it would be ``// Parsons``